bin/check-aggregate.rb

```diff
@@ -114,13 +107,6 @@ class CheckAggregate < Sensu::Plugin::Check::CLI
          long: '--pattern PATTERN',
          description: 'A PATTERN to detect outliers'
 
-  option :honor_stash,
```

Comment:
should we have `honor_silence`? I get that the api calls have been changed with the rename but I think we can still provide the same functionality right?

Reply:
OK, we just need to call that out as one of the breaking changes in the changelog then.